cg/resources/__init__.py

```diff
@@ -5,3 +5,9 @@
 valid_indexes_file = "resources/20181012_Indices.csv"
 
 valid_indexes_path = Path(pkg_resources.resource_filename("cg", valid_indexes_file))
+
+rnafusion_bundle_filenames = "resources/rnafusion_bundle_filenames.csv"
```

Comment:
```suggestion
rnafusion_bundle_filenames: str = Path("resources, "rnafusion_bundle_filenames.csv").as_posix()
```
Try to never hard code the separator on the file system, but let Path deal with it automatically. By the way could this not be a Path instead of a str (.as_posix())